flight/mcu/lib/teensyShared/dataLog.h

```diff
@@ -63,37 +68,18 @@ struct BarometerData {
     float temperature;  // in degC
     float pressure;     // in mbar
     float altitude;     // in meter
-    int32_t timeStamp_barometer;
+    systime_t timeStamp_barometer;
 };
 
 /**
  * @brief Structure for all values tracked for state estimation
  *
  */
 struct stateData {
-    float state_q0 = 0;
```

Comment:
Bro when we getting the orientation back @parthshrotri

Reply:
I actually think we could keep these variables here even if we don't use them, they'll be 0 the whole flight. But it's good incentive to improve state estimation to get orientation data.

Reply:
I'll leave it in there but commented out. Don't want to create new objects for no reason.